### ISSUE TITLE
useMaterial3 advice

### ISSUE THREAD

**Issue by fredgrott** [rydmike/flex_color_scheme#81]
We can change the do not set useMaterial3 to true advice as when those 6 widgets not yet ported get wrapped via the Material widget class you get this nice thing

[md3 demo](https://youtu.be/NQ_M_XMa4p0)

I just re worked the sdk md3 demo using FCS 6, good job!

**Comment by rydmike** [comment]
Hi @fredgrott, I'm not sure I followed exactly if this was a question or issue or just info 😄 

But yes there is still the elevation issue in Flutter 3.3 mentioned here https://github.com/flutter/flutter/issues/107190, and yes the issue can be avoided by wrapping problematic widgets, like e.g. `PopupMenuButton` and `Material` in a theme that uses Material 2. It is discussed in the issue as a potential tedious workaround.

Also in master channel there is now a change in the `Material` M3 implementation that will mostly solve the elevation in M3 for widgets that do not yet implement their M3 style and use `Material`, but it is not yet in stable channel (Flutter 3.3.0), only in master https://github.com/flutter/flutter/pull/110624, and in fact the PR is not even merged into master yet.

Yesterday I got an idea on how I could temporarily fix the themes produced by FCS, so that its M3 dialogs, drawers and popups, get the correct surface tint in M3 mode, even if it is still broken in M3 mode in Flutter 3.3.0. 

I will first check what the Flutter core team says on my question regarding when they plan to land fix https://github.com/flutter/flutter/pull/110624, if they have any plan at all to do so in a hotfix.

If it will come in a hotfix to Flutter 3.3, I will probably not implement the idea, but if the fix will not arrive until next stable, maybe Flutter 3.6.0, probably 3-4 months from now. Well then, I will add it to FCS so people can get better M3 themes already now when using M3 with FCS. 

When Flutter SDK works better on its own in M3 mode, I will remove the custom theming that fixes the issues. I've done plenty of those in the past already in FCS, that were then removed later when Flutter standard theming started working the same way, thus making my custom fixes no longer needed. Users of FCS don't notice any difference in produce result when I remove such temp fixes.
